apps/search-poc/src/containers/ResourceDetails/ResourceDetails.tsx

```diff
@@ -54,13 +59,18 @@ const ResourceDetailsContainer: React.FunctionComponent<{
   return (
     <ResourceDetails
       id={id}
+      className={Object.keys(components)
+        .map(componentName => `-${camelToKebab(componentName)}`)
```

Comment:
Why does it start with a hyphen?

Reply:
it's a variant css class 
https://rscss.io/variants.html

Reply:
wow